FirebaseStorage/Sources/Internal/StorageFetcherService.swift

```diff
@@ -84,4 +80,22 @@ actor StorageFetcherService {
     }
     response(shouldRetry)
   }
+
+  /// Map of apps to a dictionary of buckets to GTMSessionFetcherService.
+  private actor FetcherServiceMap {
+    static let shared = FetcherServiceMap()
+
+    var fetcherServiceMap: [String: [String: GTMSessionFetcherService]] = [:]
```

Comment:
nit: private if possible
```suggestion
    private var fetcherServiceMap: [String: [String: GTMSessionFetcherService]] = [:]
```